packages/core/src/common/message-translation.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+const Globalize = require("globalize");
+const supportedLocales = ["en", "es", "fr"];
+// const currentLocale = "fr";
+const localeRequire = require("../../i18n/locale.json");
+
+
+export class MessageTranslation {
+
+    localize(selection: string, defaultSelection: string): string {
+
+        try {
+            return Globalize.formatMessage(selection);
+        } catch (Error) {
+            // // If the language definition is not found, use the default selection string
+            console.log("----- using default value for: " + selection + " to be " + defaultSelection);
+            return defaultSelection;
+        }
+    }
+
+    getSupportedLocales(): {}[] {
+        return supportedLocales;
+    }
+
+    load(...lang: string[]) {
+
+        for (let i = 0; i < lang.length; i++) {
+            Globalize.loadMessages(JSON.parse(lang[i]));
+        }
+    }
+
+    // Load data translated for a specific package
+    setLocaleData(data: string) {
+        Globalize.loadMessages(JSON.parse(JSON.stringify(data)));
+    }
```

Comment:
Do not use that ... why ?

Reply:
Replace the string by an object, so simplify Globalize.loadMessages(JSON.parse(JSON.stringify(data))) by Globalize.loadMessages(data);